jobs/build_ova/build_ova.groovy

```diff
@@ -6,53 +6,55 @@ node{
     shareMethod = load("jobs/ShareMethod.groovy")
 }
 
-String label_name = "packer_ova"
-lock(label:label_name,quantity:1){
-    resources_name = shareMethod.getLockedResourceName(label_name)
-    if(resources_name.size>0){
-        node_name = resources_name[0]
-    }
-    else{
-        error("Failed to find resource with label " + label_name)
-    }
-    node(node_name){ws{
-        timestamps{
-            withEnv([
-                "RACKHD_COMMIT=${env.RACKHD_COMMIT}",
-                "RACKHD_VERSION=${env.RACKHD_VERSION}",
-                "IS_OFFICIAL_RELEASE=${env.IS_OFFICIAL_RELEASE}",
-                "OS_VER=${env.OS_VER}",
-                "BUILD_TYPE=vmware", 
-                "BINTRAY_SUBJECT=${env.BINTRAY_SUBJECT}",
-                "BINTRAY_REPO=debian",
-                "CI_BINTRAY_SUBJECT=${env.CI_BINTRAY_SUBJECT}",
-                "CI_BINTRAY_REPO=debian", 
-                "BINTRAY_COMPONENT=main", 
-                "BINTRAY_DISTRIBUTION=trusty", 
-                "BINTRAY_ARCHITECTURE=amd64"]){
-                def current_workspace = pwd()
-                deleteDir()
-                dir("on-build-config"){
-                    checkout scm
-                }
-                def url = "https://github.com/RackHD/RackHD.git"
-                def branch = "${env.RACKHD_COMMIT}"
-                def targetDir = "build"
-                shareMethod.checkout(url, branch, targetDir)
-                step ([$class: 'CopyArtifact',
-                projectName: 'PACKER_CACHE_BUILD',
-                target: 'cache_image']);
+lock("ova_build"){
```

Comment:
I think the lock can only apply to copy-artifact , aka, the build_ova can live outside the lock scope.